packages/x-tailwindcss/demo/src/components/xds-progress-bar.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <XdsBaseShowcase
+    title="Progress Bar"
+    :sections="sections"
+    #default="{ cssClass, copyCssClassesToClipboard }"
+  >
+    <div
```

Comment:
Add text for each variant like we do in the icons and such (like, for labeling it as `lg` or `md` size, the colors being used, etc.)